### ISSUE TITLE
Duplicated sniffs: Squiz.ControlStructures.ForEachLoopDeclaration.AsNotLower & Generic.PHP.LowerCaseKeyword.Found

### ISSUE THREAD

**Issue by Aliance** [squizlabs/PHP_CodeSniffer#1655]
`test.php`:
```php
<?php

function test()
{
    foreach (range(1, 10) AS $n) {
        echo $n, PHP_EOL;
    }
}
```

Running phpcs on it will trigger a duplicated errors with different sniffs. May be it should be only one?

```
phpcs --standard=PSR2 -s test.php 

FILE: /Users/aliance/www/propellerads/adp/test.php
----------------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 1 LINE
----------------------------------------------------------------------------------------------------------------------------------------
 5 | ERROR | [x] AS keyword must be lowercase; expected "as" but found "AS" (Squiz.ControlStructures.ForEachLoopDeclaration.AsNotLower)
 5 | ERROR | [x] PHP keywords must be lowercase; expected "as" but found "AS" (Generic.PHP.LowerCaseKeyword.Found)
----------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------------------------

Time: 46ms; Memory: 4Mb
```

**Comment by gsherwood** [comment]
Sorry, finally getting around to this issue.

I agree, the PSR2 standard doesn't need to report two errors here, so I've excluded `Squiz.ControlStructures.ForEachLoopDeclaration.AsNotLower`.

Thanks for the suggestion.
